perma_web/perma/templates/email/stats.html

```diff
@@ -3,21 +3,17 @@
 
 ## The Folks
 {{ num_regular_users_added }} regular users added. From {{ prev_regular_users_count }} to {{ current_regular_users_count }}
-{{ num_vesting_members_added }} vesting users added. From {{ prev_vesting_members_count }} to {{ current_vesting_members_count }}
+
```

Comment:
Yup. I'll add that back  in. The stats page and email need a bit of reworking after the model changes (next pull req).